improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -792,9 +822,9 @@ def test_basic(self):
         plugin = WeatherSymbols()
         result = plugin.process(self.cubes)
         self.assertIsInstance(result, iris.cube.Cube)
-        self.assertArrayEqual(result.attributes["weather_code"], self.wxcode)
+        self.assertArrayAndMaskEqual(result.attributes["weather_code"], self.wxcode)
```

Comment:
Don't think this attribute would ever be masked - suggest reverting to original call for clarity.

Reply:
Agreed, and updated.